lib/server/websocket.js

```diff
@@ -405,20 +406,22 @@ function init (env, ctx, server) {
               }
               // if not found create new record
               console.log(LOG_DEDUP + 'Adding new record');
-              ctx.store.collection(collection).insert(data.data, function insertResult (err, doc) {
+              ctx.store.collection(collection).insertOne(data.data, function insertResult (err, ops) {
                 if (err != null && err.message) {
                   console.log('treatments data insertion error: ', err.message);
                   return;
                 }
 
+                var doc = data.data;
```

Comment:
Do you have a roadmap or anything around browser support? How much longer will the codebase use `var` instead of let/const?

Looks like any android/ios device since 2011 should support let/const - there are no vendor-currently-supported os/browser combinations that don't support them.

Maybe the approach could be "use nightscout 15 for old browsers" and do any work on the f/e to support newer testing/frameworks in a nightscout v16 that will only support newer browsers? I think you'll rapidly paint yourself into a corner if "we're not waiting" becomes "we support your ancient devices forever"